torch/csrc/jit/register_special_ops.cpp

```diff
@@ -0,0 +1,30 @@
+#include "torch/csrc/autograd/profiler.h"
+#include "torch/csrc/jit/custom_operator.h"
+#include "torch/csrc/jit/operator.h"
+
+namespace torch {
+namespace jit {
+
+namespace {
+RegisterOperators reg({
+    Operator(
+        "aten::split(Tensor self, int[] split_sizes, int dim=0) -> Tensor[]",
+        [](Stack& stack) {
+          autograd::profiler::RecordFunction record("split_with_sizes");
+          auto result = at::split_with_sizes(
+              (std::move(peek(stack, 0, 3))).toTensor(),
+              (std::move(peek(stack, 1, 3))).toIntList()->elements(),
+              (std::move(peek(stack, 2, 3))).toInt());
+          drop(stack, 3);
+          pack(stack, std::move(result));
+          return 0;
+        }),
+    Operator(
+        "aten::Size(int[] sizes) -> int[]",
```

Comment:
What does this operator do? When would we instantiate it?

Reply:
It's in some of the [`test_autograd.py` tests](https://github.com/pytorch/pytorch/blob/master/test/test_autograd.py#L2711) as `torch.Size`. There's some discussion about it in #8520.